cli/provider/core_service_linux.go

```diff
@@ -36,7 +39,7 @@ func Get(ctx context.Context, cmd string, cfg *config.Config, logger *zap.Logger
 	if err != nil {
 		return nil, err
 	}
-
+	contractSvc := contract.New(logger, commonServices.YamlTestDB, commonServices.YamlMockDb, commonServices.YamlOpenAPIDb, cfg)
```

Comment:
Since the keploy contract feature is not dependent on the operating system you should also add service in the `core_service_others.go` file.